samza-azure/src/main/java/org/apache/samza/AzureLeaderElector.java

```diff
@@ -107,5 +107,4 @@ public boolean amILeader() {
   public LeaseBlobManager getLeaseBlobManager() {
     return this.leaseBlobManager;
   }
-
 }
```

Comment:
Can you please add a log line for the `else` condition above Line#73 ? 
I don't remember if we discussed this before. You should throw an exception when there is an `AzureException`.